src/main/kotlin/omok/model/game/OmokGame.kt

```diff
@@ -0,0 +1,52 @@
+package omok.model.game
+
+import omok.model.stone.Coordinate
+import omok.model.stone.GoStone
+import omok.model.stone.GoStoneColor
+
+class OmokGame(private val board: Board) {
+    private val blackRenjuRuleAdapter: BlackOmokRuleAdapter = BlackOmokRuleAdapter(board)
+    private val whiteRenjuRuleAdapter: WhiteOmokRuleAdapter = WhiteOmokRuleAdapter(board)
```

Comment:
```suggestion
    private val blackRenjuRuleAdapter: OmokRuleAdapter = BlackOmokRuleAdapter(board)
    private val whiteRenjuRuleAdapter: OmokRuleAdapter = WhiteOmokRuleAdapter(board)
```
가급적이면 부모타입 혹은 인터페이스를 타입을 활용해보면 어떨까요?
위와 같이 사용하게 되면 프로그램이 훨씬 유연해질 거예요. 
인터페이스와 다형성에 대해 알아보고 아래의 글도 참고하면 좋을 것 같아요.

ref. https://stackoverflow.com/questions/11683044/return-arraylist-or-list/11683073